Formula/m/minizip-ng.rb

```diff
@@ -32,18 +33,20 @@ class MinizipNg < Formula
   conflicts_with "libtcod", "libzip", because: "libtcod, libzip and minizip-ng install a `zip.h` header"
 
   def install
-    system "cmake", "-S", ".", "-B", "build/shared",
-                    "-DMZ_FETCH_LIBS=OFF",
-                    "-DBUILD_SHARED_LIBS=ON",
-                    *std_cmake_args
+    args = %w[
+      -DMZ_FETCH_LIBS=OFF
+      -DMZ_LIB_SUFFIX=-ng
+      -DMZ_LIBCOMP=OFF
+      -DMZ_ZLIB=ON
```

Comment:
Suffix is also added by Arch (https://gitlab.archlinux.org/archlinux/packaging/packages/minizip-ng/-/blob/main/PKGBUILD?ref_type=heads#L23) and Nix (https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/libraries/minizip-ng/default.nix#L34)

`MZ_LIBCOMP` is used as alternative to `MZ_ZLIB` (https://github.com/zlib-ng/minizip-ng/blob/4.0.3/CMakeLists.txt#L164-L165) so need to disable as `opencolorio` has hard requirement on `zlib` support https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/main/docs/quick_start/installation.rst?plain=1#L235-L240

> Please note that if you provide your own ``minizip-ng``, rather than having OCIO's CMake
download and build it, you will likely need to set its CMake variables the same way
that OCIO does (e.g., enable ZLib and turn off most other options).  Using a ``minizip-ng``
from various package managers (e.g., Homebrew) probably won't work.  Please see the
settings that begin with ``-DMZ_`` that are used in the OCIO 
`minizip-ng find module. <https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/main/share/cmake/modules/Findminizip-ng.cmake>`_